docs/how-to-build.md

```diff
@@ -26,7 +26,9 @@ This software is licensed under the Apache License version 2."
   You don't have to preinstall the specified JDK environment, as Gradle detects the JDK version needed and downloads it automatically.
 + Gravitino uses the Gradle Java Toolchain to detect and manage JDK versions, it checks the
   installed JDK by running the `./gradlew javaToolchains` command. See [Gradle Java Toolchain](https://docs.gradle.org/current/userguide/toolchains.html#sec:java_toolchain).
-+ Make sure you have installed Docker in your environment if you plan to run integration texts. Without it, some Docker-related tests may not run.
++ Gravitino excludes all Docker-related tests by default. To run integration tests, make sure you have installed
+  Docker in your environment and set `skipDockerDependentTests=false` in the `gradle.properties` file(or
+  use `-PskipDockerDependentTests=false` in the command). Otherwise, some Docker-related tests may not execute.
```

Comment:
What do we do with the configuration `skipITs`? Do we need to remove it?

Reply:
I don't think we need to do anything with `skipITs`. Because both UT and IT may use Docker, parameter `skipITs` is used to control IT, while the new configuration `skipDockerDependentTests` is used to control Docker.

Reply:
I understand what you mean. Do other people have any suggestions about it?

Reply:
I prefer to combine `skipITs` and `skipDockerDependentTests`, because it's too complicated.

Reply:
> I prefer to combine `skipITs` and `skipDockerDependentTests`, because it's too complicated. maybe we could use `skipTestsWithDocker` and `skipTestsWithoutDocker`

This is the same with removing `skipITs `, right?